Created Github Organization Membership page, completed #2197 (#2284)
meshery/meshery.io: collections/_handbook/instructions.md

---
layout: page
title: GitHub Organization Membership
description: Consistent contributors may receive invitations to Meshery GitHub organizations.
---

Membership in the GitHub organizations is a significant milestone for contributors who have shown persistent commitment and dedication to Meshery projects.

It’s not solely about writing code — it's about consistent engagement and alignment with the community’s values.

---

## Membership Requirements

> Contributors who may be invited typically:

Reviewer: ```suggestion
Membership to the GitHub organizations is a significant milestone for contributors who have shown persistent commitment and dedication to Meshery projects.
It is not solely about writing code but rather the consistency of engagement and alignment of mentality.

Contributors who may be invited typically:
```

- Have been actively involved in the community for several weeks
- Make regular updates to projects
- Help other community members
- Attend development meetings
- Demonstrate a willingness to learn and share knowledge
- Show a genuine desire to improve themselves, others, and the projects

---

## Invitation Process

Depending on the **number and quality of contributions** across the five GitHub organizations under Meshery, contributors may receive an invitation to one or more of these organizations.

The process includes:

1. Starting a group DM
2. Executing a Slack slash command in `#community-management`

---

## Group Message for Invitations

> When you identify a worthy contributor:  
> 📥 Start a group DM with the contributor and other maintainers or appropriate members.

Reviewer: ```suggestion
When you identify a worthy contributor, send a message in a group DM that includes the individual and other maintainers/appropriate members.
```

The content can be copied as is from the reference page @Flashl3opard

---

## Sending the GitHub Invitation

Use the following Slack command in the `#community-management` channel:  

> `/invite-github [email address] [organization name]`

Reviewer: ```suggestion
```/invite-github [email address] [organization name]```
```

> 

Reviewer: ```suggestion
Where [organization name] should be:

- meshery – to invite the email address to Meshery GitHub organization under the "contributors" team
- meshery-extensions – to to invite the email address to Meshery Extensions GitHub organization under the "contributors" team

For example:

<div class="highlight-code">
<code id="invite-github" class ="code-box" style="white-space:pre-wrap;"> /invite-github contributor@example.com meshery</code>
{% include copy-to-clipboard.html clipboard_target="#git-add" clipboard_text="/invite-github contributor@example.com meshery" %}
</div>
```

Reviewer: @Flashl3opard can you incorporate this suggestion?

1. Add Meshery-Extensions to the organizations list
2. use the codeblock for the command

Reviewer: @Flashl3opard 

I meant for you to use this code block, the standard one looks a bit too bright.

```
<div class="highlight-code">
<code id="invite-github" class ="code-box" style="white-space:pre-wrap;"> /invite-github contributor@example.com meshery</code>
{% include copy-to-clipboard.html clipboard_target="#git-add" clipboard_text="/invite-github contributor@example.com meshery" %}
</div>
```

LGTM, otherwise 👍🏼
